lib/JAHSBench/README.md

```diff
@@ -1,3 +1,109 @@
-# JAHS-Bench-201
+# JAHS Benchmark Suite
 
-* [JAHS-Bench-201 - Github](https://github.com/automl/jahs_bench_201)
\ No newline at end of file
+This module contains a DeepHyper wrapper for
+ [JAHS-Bench-201](https://github.com/automl/jahs_bench_201).
+
+JAHSBench implements a random forest surrogate model, trained on real-world
```

Comment:
JAHS is using XGBoost I think: https://openreview.net/pdf?id=_HLcjaVlqJ
Section 2.3, page 4